neo/conf/application.conf

```diff
@@ -76,6 +76,7 @@ bounded-mailbox {
 }
 
 neo.timeout.initial = "10 minutes"
+neo.stream.max.frame.length = "8388608 Bytes"
```

Comment:
This is for maximum 8MB per chunk. I am not very sure of the number. It cannot be too small to hold a single piece of data returned by Berry, nor can be so large that will take memory and time to transport. Maybe we need experiments to find the optimal value. In short, I need suggestions on that.